book/garbage-collector/README.md

```diff
@@ -573,13 +573,13 @@ Benchmark for mutable, immutable
   barrier_bench.exe [COLUMN ...]
 
 Columns that can be specified are:
-	time       - Number of nano secs taken.
-	cycles     - Number of CPU cycles (RDTSC) taken.
-	alloc      - Allocation of major, minor and promoted words.
-	gc         - Show major and minor collections per 1000 runs.
-	percentage - Relative execution time as a percentage.
-	speedup    - Relative execution cost as a speedup.
-	samples    - Number of samples collected for profiling.
```

Comment:
ah, did you accidentally tab-to-space this output, or deliberately?

Reply:
Ugh, sorry. Not on purpose. My emacs is configured to untabify-on-save. Sorry!

Reply:
no worries; reverted in 4ba4ec0eea88aa2fe053e0a77dd09cc13e03a78f and CI should be green soon